bootstrap.php

```diff
@@ -0,0 +1 @@
+<?php require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
can be placed in test folder, example: https://github.com/ezsystems/repository-forms/blob/master/tests/bootstrap.php

Reply:
yes, but possibly we can have tests also in /src/bundle directory, then the file would be duplicated